ddtrace/contrib/trace_utils.py

```diff
@@ -102,7 +102,9 @@ def _store_headers(headers, span, integration_config, request_or_response):
 
     for header_name, header_value in headers.items():
         tag_name = integration_config._header_tag_name(header_name)
-        if tag_name is None:
+        if header_name.lower() == "user-agent":
```

Comment:
we'll want to performance test this change just to be on the safe side, since this will get called for every header on every request